lib/layer.js

```diff
@@ -113,14 +122,23 @@ Layer.prototype.match = function match(path) {
     return false
   }
 
-  if (this.regexp.fast_slash) {
+  if (this.fastSlash) {
     // fast path non-ending match for / (everything matches)
     this.params = {}
     this.path = ''
+    this.matchedPath = this.paths[0]
     return true
   }
 
-  var m = this.regexp.exec(path)
+  var checkPath, m;
```

Comment:
All of the express modules are working toward compliance with `standard`.  Which includes the rule that variable declarations are each on a single line.  We have doing it piecemeal because it would cause too many conflicts otherwise, so since this is a new line, can you make the change just here?

Reply:
Fixed in https://github.com/pillarjs/router/pull/86/files#diff-be883fbed8968009576c828e0ff70e36R121